<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>7</number>
   </property>
   <item row="2" column="1" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>wireless</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="2,1,1">
      <item row="0" column="1">
       <widget class="QLineEdit" name="devicePortEdit">
        <property name="placeholderText">
         <string>8080</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="deviceIpEdit">
        <property name="placeholderText">
         <string>192.168.0.103</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="wirelessButton">
        <property name="text">
         <string>wireless</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QTextEdit" name="outEdit"/>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>350</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="closeButton">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: rgba(128, 128, 128, 0.1);
}
QPushButton:pressed {
    background-color: rgba(128, 128, 128, 0.2);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/close.png</normaloff>:/res/close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QGroupBox" name="topGroupBox">
     <property name="title">
      <string>USB Line</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>  device serialï¼š</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="StartServerButton">
        <property name="text">
         <string>Start Server</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QLineEdit" name="SerialEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="getIpButton">
        <property name="text">
         <string>get ip</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="startAdbButton">
        <property name="text">
         <string>start device adbc</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="updateDeviceButton">
        <property name="text">
         <string>device update</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="StopServerButton">
        <property name="text">
         <string>StopServer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="minButton">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: rgba(128, 128, 128, 0.1);
}
QPushButton:pressed {
    background-color: rgba(128, 128, 128, 0.2);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/min5.png</normaloff>:/res/min5.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="inforButton">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: rgba(128, 128, 128, 0.1);
}
QPushButton:pressed {
    background-color: rgba(128, 128, 128, 0.2);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/infor.png</normaloff>:/res/infor.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
